gazebo_color_plugin/launch/main.launch

```diff
@@ -0,0 +1,8 @@
+<launch>
+  <!-- We resume the logic in empty_world.launch, changing only the name of the world to be launched -->
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <arg name="verbose" value="true"/>
+    <arg name="world_name" value="$(find gazebo_light_sensor_plugin)/worlds/light.world"/>
```

Comment:
Did you create another world? the idea is to use the color sensors in the autorace one.

Reply:
Adressed on the PR referenced below